src/lib/geojson.js

```diff
@@ -22,3 +22,8 @@ export const mapPolygonRecordToGeoJson = polygon =>
     name: polygon[DB_FIELDS.PROPERTIES_NAME],
     propertyType: polygon[DB_FIELDS.PROPERTIES_TYPE]
   });
+
+export const formatFeaturesToFeatureCollection = ({ features = [] }) => ({
+  type: "FeatureCollection",
```

Comment:
Lets make this a const so everyone is encouraged to use it and the change of someone making a typo is reduced

Reply:
gotchya